javaparser-core/src/main/java/com/github/javaparser/metamodel/SwitchEntryMetaModel.java

```diff
@@ -42,6 +42,8 @@ public class SwitchEntryMetaModel extends NodeMetaModel {
         super(superBaseNodeMetaModel, SwitchEntry.class, "SwitchEntry", "com.github.javaparser.ast.stmt", false, false);
     }
 
+    public PropertyMetaModel isDefaultPropertyMetaModel;
```

Comment:
Did you see this comment "This file, class, and its contents are completely generated based"?

Reply:
This change is generated code